internal/api/device.go

```diff
@@ -153,7 +153,8 @@ type DevicesResponse struct {
 
 //DeviceACLTemplateRequest describes request for updating device ACL template
 type DeviceACLTemplateRequest struct {
-	TemplateUUID *string `json:"aclTemplateUuid"`
+	TemplateUUID        *string `json:"aclTemplateUuid,omitempty"`
+	MgmtAclTemplateUuid *string `json:"mgmtAclTemplateUuid,omitempty"`
```

Comment:
`MgmtAclTemplateUuid` should have `UUID` capitalized to match `TemplateUUID`.
I should have caught this before merging.

Reply:
`ID`/`UUID` appears to be the convention we've been using in `ne-go`. We have many examples of `...UUID`, `...ID`, `IP...`, and `MAC...`.  

We have some exceptions like  `SshInterfaceId`, `ClusterId`, and `LicenseFileId`. Because these are in `internal/` we can change these freely.

It looks like we should update https://github.com/equinix/ne-go/pull/12/files#diff-79aab787cbc1ff2a0a825a43f479b275e28ea9cc070c72eeede7aeda30894c25R88 and https://github.com/equinix/ne-go/pull/12/files#diff-79aab787cbc1ff2a0a825a43f479b275e28ea9cc070c72eeede7aeda30894c25R69 to use the `MgmtAclTemplateUUID` too. These are inside of the `internal` path, so there is no breaking change. 

We also have some public exceptions:
```
client.go:262:	WanInterfaceId      *string
client.go:434:	ClusterId          *string
client.go:442:	LicenseFileId       *string.
```

Perhaps we can standardize those in a future _breaking_ change.